Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -398,3 +412,37 @@ extension BrowserViewController: LongPressGestureDelegate {
         self.presentViewController(actionSheetController, animated: true, completion: nil)
     }
 }
+
+extension BrowserViewController : Transitionable {
+    func showTransition(options: TransitionOptions) {
```

Comment:
Sorry, the refactoring PR probably bitrotted some of this. To stay consistent with the delegate pattern we're using everywhere now, could you rename these?
```
func transitionableWillShow(transitionable: Transitionable, options: TransitionOptions)
func transitionableWillHide(transitionable: Transitionable, options: TransitionOptions)
func transitionableWillComplete(transitionable: Transitionable, options: TransitionOptions)
```